### DISCUSSION TITLE
Inquiries about wincount for multiple minor

### DISCUSSION THREAD

**Discussion by scvpark** [filecoin-project/lotus#6101] [Storage Provider] [answered]
lotus version 1.6.0+mainnet+git.3fc23a785

I running 3 Miners in one public IP.
Each minor is running on a different Port.
Is there any conflict to get winning count when operating multiple miners on one IP?
Or is there any issue related to wincount in Lotus 1.6 version?

**Comment by Leozhang404** [comment]
Maybe you will be punished because of consensus faults.

**Comment by SgtCoin** [comment, marked as answer]
Archiving for future reference due to age. Network is now on v 1.13+
